app/src/forms/form/service.js

```diff
@@ -120,6 +120,40 @@ function isClosingMessageValid(schedule) {
 }
 
 const service = {
+  /**
+   * Validates reminder settings against schedule configuration
+   * Ensures reminders are only enabled when a valid schedule exists
+   * @param {Object} data Form data containing schedule and reminder_enabled
+   * @returns {boolean} Validated reminder_enabled value
+   */
+  _validateReminderSettings: (data) => {
+    // If reminders are not enabled, no validation needed
+    if (!data.reminder_enabled) {
+      return false;
+    }
+
+    // Check if schedule exists and is properly configured
+    const hasValidSchedule =
+      data.schedule &&
+      data.schedule.enabled === true &&
+      data.schedule.scheduleType !== null &&
+      data.schedule.scheduleType !== undefined &&
+      data.schedule.scheduleType !== ScheduleType.MANUAL;
+
+    if (!hasValidSchedule) {
+      return false;
+    }
+
+    // Additional check: ensure open date exists and is in the future
+    if (data.schedule.openSubmissionDateTime) {
```

Comment:
We should be leveraging logic in scheduleService. You spent a lot of time working with schedules and dates to make sure the timezones were respected and there are functions for doing all the date checks. Use that functionality.

Also, add tests for this function in the unit test class.